src/Microsoft.ML.Data/Evaluators/MultiOutputRegressionEvaluator.cs

```diff
@@ -547,7 +547,7 @@ private protected override Delegate[] CreateGettersCore(Row input, Func<int, boo
 
             var t = schema[LabelIndex].Type as VectorType;
             if (t == null || !t.IsKnownSize || (t.ItemType != NumberType.R4 && t.ItemType != NumberType.R8))
-                throw Host.Except("Label column '{0}' has type '{1}' but must be a known-size vector of R4 or R8", LabelCol, t);
+                throw Host.Except(nameof(schema), "label", LabelCol, "known-size vector of float or double", t.ToString());
```

Comment:
i'd keep the word "type":

"known-size vector with items of type float or double"